app/controllers/static_pages_controller.rb

```diff
@@ -5,6 +5,7 @@ class StaticPagesController < ApplicationController
   layout :layout_for_page
 
   def show
+    @games = Game.all
```

Comment:
This should go in the games controller in def index. def index in the games controller is the home page for when the user is logged in. static_pages#show, id='home' is the home page before the user logs in :)